docs/cloud-native-security/d4c-policy-guide.asciidoc

```diff
@@ -0,0 +1,94 @@
+| containerImageName | A list of container image names to match on. For example: `nginx`.
+| containerImageTag | A list of container image tags to match on. For example: `latest`.
+| kubernetesClusterId | A list of Kubernetes cluster IDs to match on. For consistency with KSPM, the `kube-system` namespace's UID is used as a cluster ID.
+| kubernetesClusterName | A list of Kubernetes cluster names to match on.
+| kubernetesNamespace | A list of Kubernetes namespaces to match on.
+| kubernetesPodName | A list of Kubernetes pod names to match on. Trailing wildcards supported.
+| kubernetesPodLabel | A list of resource labels. Trailing wildcards supported (value only), for example: `key1:val*`.
+|===
+
+[discrete]
+=== File-selector conditions
+These conditions are available only for `file` selectors.
+
+[cols="1,1", options="header"]
+|===
+| Name | Description
+| operation | The list of system operations to match on. Options include `createExecutable`, `modifyExecutable`, `createFile`, `modifyFile`, `deleteFile`.
+| ignoreVolumeMounts | If set, ignores file operations on ALL volume mounts.
+| ignoreVolumeFiles | If set, ignores operations on file mounts only. For example: mounted files, `configMaps`, and secrets.
+| targetFilePath | A list of file paths to include. Paths are absolute and wildcards are supported. The * wildcard matches any sequence of characters within a single directory, while the ** wildcard matches any sequence of characters across multiple directories and subdirectories. Note: In order to ensure precise targeting of file integrity monitoring operations, a `TargetFilePath` is required whenever the `deleteFile`, `modifyFile`, or `createFile` operations are used within a selector.
+|===
+
+NOTE:
+
+[discrete]
+=== Process-selector conditions
+These conditions are available only for `process` selectors.
+
+[cols="1,1", options="header"]
+|===
+| Name | Description
+| operation | The list of system operations to match on. Options include `fork` and `exec`.
+| processExecutable | A list of executables (full path included) to match on. For example: `/usr/bin/cat`. Wildcard support is same as targetFilePath above.
+| processName | A list of process names (executable basename) to match on. For example: `bash`, `vi`, `cat`.
+| sessionLeaderInteractive | If set to true, will only match on interactive sessions (i.e. sessions with a controlling TTY).
+|===
+
+[discrete]
+=== Response fields
+A policy can include one or more responses. Each response is comprised of the following fields:
+
+[cols="1,1", options="header"]
+|===
+| Field | Description
+| match | An array of one or more selectors of the same type (`file` or `process`).
+| exclude | Optional. An array of one or more selectors to use as exclusions to everything in `match`.
+| actions | An array of actions to perform when at least one `match` selector matches and none of the `exclude` selectors match. Options include `log`, `alert`, and `block`. See *Response actions*, below.
```

Comment:
This isn't really how we handle cross-references, and also avoid "See" in favor of "Refer to," etc. But maybe you could just omit the reference, since the actions are immediately explained in the next section?
```suggestion
| actions | An array of actions to perform when at least one `match` selector matches and none of the `exclude` selectors match. Options include `log`, `alert`, and `block`.
```